src/styles/blocks/bike.scss

```diff
@@ -0,0 +1,56 @@
+.bike {
+  &__photo-container {
+    position: relative;
+    width: 100%;
+    max-height: 320px;
+    overflow: hidden;
+    transition: transform 0.3s;
+
+    &:hover {
+      transform: scale(1.05);
+    }
+
+    &::after {
+      content: "";
+      display: block;
+      padding-bottom: 100%;
+    }
+
+    @include tablet {
+      height: 186px;
+      padding-bottom: 0;
+    }
+
+    @include desktop {
+      height: 320px;
+      padding-bottom: 0;
+    }
+  }
+
+  &__photo {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+
+    object-fit: cover;
+    object-position: center;
+  }
+
+  &__title {
+    font-family: "Poppins", sans-serif;
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 140%;
+    color: $first-text-colour;
+
+    padding: 16px 0 8px;
+  }
+
+  &__description {
+    font-family: "Poppins", sans-serif;
+    font-weight: 400;
+    font-size: 16px;
```

Comment:
You use `font-size: 16px` too often. It be better to add this property to entire page